TestPlan/InstallAsNew

```diff
@@ -0,0 +1,16 @@
+#Test Plan for Ensuring that Installing as an Update works properly
```

Comment:
These files don't have a file extension, so GitHub doesn't know they are Markdown files. Can you change these to have a *.md file extension?